### ISSUE TITLE
Frame Processor Error: Regular javascript function '' cannot be shared. Try decorating the function with the 'worklet' keyword to allow the javascript function to be used as a worklet., js engine: VisionCamer

### ISSUE THREAD

**Issue by tomhopkins** [tony-xlh/vision-camera-dynamsoft-label-recognizer#4]
Hi, thanks very much for this pluggin.

I'm persistently getting: 

Frame Processor Error: Regular javascript function '' cannot be shared. Try decorating the function with the 'worklet' keyword to allow the javascript function to be used as a worklet., js engine: VisionCamera

When running the example code on a dev build (expo).

Any pointers would be greatly appreciated.

Tom

**Comment by xulihang** [comment]
What is your frame processor function look like? 

I've updated the example to vision camera 3.9.0. You can try to update it and try again.

**Comment by tomhopkins** [comment]
Thanks, It was the anonymous function that was triggering it I think. I ended up copying your MRZ specific example (https://github.com/tony-xlh/react-native-mrz-scanner) and it compiled. I actually had a lot of trouble getting it to recognise MRZ (perhaps to do with model and custom template). So I've ended up creating my demo another way. Will come back to Dynamasoft if we take the feature into production.
